src/configure/configure.ts

```diff
@@ -254,7 +255,7 @@ class Orchestrator {
     }
 
     private async analyzeNode(node: any): Promise<GenericResource> {
-        if (!!node && !!node.fullId) {
+        if (!!node) {
```

Comment:
this might break "right click on folder" as node will not be null, but no azure resource will be found. I think you should just call `this.extractAzureResourceFromNode(node)` always and let that function return true/false depending on whether it found a resource. If false, then go to L262.